src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,24 @@
+package racinggame.domain;
+
+import racinggame.dto.SnapShotDto;
+import racinggame.random.Radom;
+
+public class RacingGame {
+
+    private Cars cars;
+    private Integer repeatCount;
+
+    public RacingGame(int repeatCount,int carCount) {
+        this.repeatCount=repeatCount;
+        this.cars = Cars.of(carCount);
+    }
+
+
+    public SnapShotDto start(Radom radom) {
```

Comment:
```suggestion
    public RacingGame(int repeatCount, int carCount) {
        this.repeatCount=repeatCount;
        this.cars = Cars.of(carCount);
    }

    public SnapShotDto start(Radom radom) {
```
사소해보여도 띄어쓰기와 개행은 지켜주면 좋을 것 같아요 😄